core/scripts/server.js

```diff
@@ -120,6 +105,29 @@ app.get('/invalidate', (req, res) => {
   } else {
     utils.apiStatus(res, 'Cache invalidation is not required, output cache is disabled', 200)
   }
+}
+
+const serve = (path, cache, options) => express.static(resolve(path), Object.assign({
+  maxAge: cache && isProd ? 60 * 60 * 24 * 30 : 0
+}, options))
+
+const themeRoot = require('../build/theme-path')
+
+app.use('/dist', serve('dist', true))
+app.use('/assets', serve(themeRoot + '/assets', true))
+app.use('/service-worker.js', serve('dist/service-worker.js', {
+  setHeaders: {'Content-Type': 'text/javascript; charset=UTF-8'}
+}))
+
+const serverExtensions = require(resolve('src/server'))
+serverExtensions.registerUserServerRoutes(app)
+
+app.post('/invalidate', (req, res) => {
+  invalidateCache(req, res)
+})
+
```

Comment:
It will be a little bit nicer:
app.post(‘/invalidate’, invalidateCache)
app.get(‘/invalidate’, invslidsteCache)

Reply:
Haha, I actually did this already but forgot to push =) I will do this now.